benches/test_ml_server.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
Can you 
1: Write a line or two about how to run this (cause I assume you have to run it with pytest/profile to have the `benchmark.pedantic` fixture)
2: Change the folder to e.g benchmarks
3: Put a single line comment above the "benchmarks" fixture saying its a pytest fixture, otherwise it is just some magic shit which is just there